package.json

```diff
@@ -8,10 +8,11 @@
   "repository": "https://github.com/keystonejs/keystone",
   "homepage": "https://github.com/keystonejs/keystone",
   "scripts": {
+    "rune": "manypkg run",
```

Comment:
I think I'm -1 on this, mostly because it adds one more layer of abstraction for people to understand without really giving us any significant wins.

Reply:
Since one of the complaints about was amount of typing. Will remove but would also appreciate an alternative.